### DISCUSSION TITLE
一直提示Authentication servers are down. Please try again later, sorry!

### DISCUSSION THREAD

**Discussion by Abandoned9** [bs-community/blessing-skin-server#321] [General]
spigot 1.17.1
6.0.0 beta3
/api/yggdrasil能够正常访问，启动器也能验证账号密码
但是在进入服务器的过程中会提示Authentication servers are down. Please try again later, sorry!
并且是一直提示

**Comment by g-plane** [comment]
日志呢？

**Reply by Abandoned9** [reply]
`[00:39:07] [User Authenticator #4/ERROR]: Couldn't verify username because servers are unavailable
[00:39:07] [Server thread/INFO]: com.mojang.authlib.GameProfile@71f74773[id=<null>,name=DimitriYoon,properties={},legacy=false] (/180.162.25.147:10305) lost connection: Authentication servers are down. Please try again later, sorry!
`

**Reply by g-plane** [reply]
有 BS 相关的日志吗？

**Reply by Abandoned9** [reply]
没有，我查过，一直很正常

**Reply by Abandoned9** [reply]
皮肤站服务器和游戏服务端是不同的服务器，皮肤站和验证服务器都很正常，但是我的游戏端服务器，是屏蔽了http和https请求的，是否是这个原因呢？

**Reply by Abandoned9** [reply]
这个验证是否需要对游戏端进行http或者https请求呢？

**Reply by g-plane** [reply]
需要的。
